apps/prairielearn/src/ee/pages/instructorAiGenerateJobs/instructorAiGenerateJobs.sql

```diff
@@ -0,0 +1,16 @@
+-- BLOCK select_generation_sequence_by_course
+SELECT
+  j.*,
+  u.email
```

Comment:
As requested in https://github.com/PrairieLearn/PrairieLearn/pull/10301#discussion_r1700549115, please use the user UID. `email` is not guaranteed to be populated and isn't something we'd use to identify a user to another user.

Consider structuring the query like this instead:

```sql
SELECT to_jsonb(j.*) AS job, to_jsonb(u.*) AS user ...
```

This will allow you to structure the corresponding schema in TS as this:

```ts
const JobRowSchema = z.object({
  job: JobSchema,
  user: UserSchema,
});
```

This avoids the creation of even more bespoke types like `JobWithUserEmailSchema`.

Reply:
Done in deabe2646069d6cce9d383460ea361768ab6c66c